### DISCUSSION TITLE
How to stop extension from providing error squiggles

### DISCUSSION THREAD

**Discussion by SirCupOfReese** [microsoft/vscode-cmake-tools#3383] [Q&A]
I recently switched to clangd for code completion and error squiggles in vscode and ran into an issue where I was getting error squiggles from multiple sources.

<img width="402" alt="image" src="https://github.com/microsoft/vscode-cmake-tools/assets/21353072/07bbc2fa-630e-481b-9639-6581a361d220">

The second error squiggle is, seemingly, from clangd. The first one, however, is not. It lists the error source from GCC and will not go away even if I resolve the error. I have to manually rebuild for the error to leave, whereas the issue listed from Clang updates regularly and disappears after fixing the line.

I did some Googling and found [this](https://github.com/clangd/vscode-clangd/issues/444) GitHub issue, which seems to state that CMake Tools is somehow providing error squiggles. I tried disabling the extension momentarily, and lo and behold, the clang error was the only one present. I need to keep the CMake Tools extension enabled (I guess I don't *need* to, but I'd **really** like to) but it's getting very frustrating having to rebuild to see if my error is resolved, especially since I know that the squiggles *would* be working correctly if CMake Tools wasn't adding lasting squiggles to the pile.

Has anyone experienced this? If not, any suggestions? I'd love to be able to use this extension.

Thank you for any help.

**Comment by gcampbell-msft** [comment]
Based on the linked GitHub issue, this should be resolved. Closing this dicussion.

If not, please feel free to open another discussion post or preferably a GitHub issue if there is a possible bug. Thanks!
